service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -46,21 +46,21 @@ public static DeviceTwin createFromConnectionString(String connectionString) thr
     /**
      * Static constructor to create instance from connection string.
      *
-     * @param connectionString The iot hub connection string.
-     * @param options the configurable options for each operation on this client. May not be null.
+     * @param connectionString The IoT hub connection string.
```

Comment:
I have observed IoT **H**ub in ms docs and also in some C# headers.

Reply:
The standard they communicated about a year ago is, when talking about a hub instance, to use a lowercased 'h'. When talking about the service, you'd use Title Casing.